src/System.Private.Uri/tests/FunctionalTests/IriRelativeFileResolutionTest.cs

```diff
@@ -45,8 +47,8 @@ public void IriRelativeResolution_CompareImplcitAndExplicitFileWithReservedChar_
         [Fact]
         public void IriRelativeResolution_CompareImplcitAndExplicitFileWithUnicodeIriOn_AllPropertiesTheSame()
         {
-            string unicodeImplicitTestFile = @"c:\path\\u30AF\path3\\u30EB\u30DE.text";
-            string nonUnicodeImplicitFileBase = @"c:\path\file.txt";
+            string unicodeImplicitTestFile = s_isWindowsSystem ? @"c:\path\\u30AF\path3\\u30EB\u30DE.text" : "/path//u30AF/path3//u30EB/u30DE.text";
```

Comment:
I have to say, I'm confused by this test.  What's the point of the \u30AF etc stuff if it's not meant to be a Unicode char?